DataFormats/HcalDetId/src/HcalCalibDetId.cc

```diff
@@ -33,6 +33,18 @@ HcalCalibDetId::HcalCalibDetId(int ieta, int iphi) : HcalOtherDetId(HcalCalibrat
       |(((ieta > 0)?(1):(0))<<11); // z side, bit [11]
 }
 
+HcalCalibDetId::HcalCalibDetId(CalibDetType dt, int value) : HcalOtherDetId(HcalCalibration) {
+  id_|=(dt<<17);
+  id_|=value&0xFF;
```

Comment:
I just wanted to make sure that this value is not changing event by event to give, say, readout values (ADC or something like that, which wouldn't belong in here)
... DetId is supposed to correspond to a fixed geometric unit. HcalCalibDetId appears to violate this already at least partly. Oh, well.